react-components/src/components/CadModelContainer/useApplyCadModelStyling.tsx

```diff
@@ -99,32 +99,54 @@ async function applyStyling(
   }
 }
 
-function isEqualStylingGroupAndCollection(
+async function tryUpdateStylingGroup(
```

Comment:
Still unsure about this name - could we have something that includes e.g. `checkSimilarity...`?